tests/test_landings.py

```diff
@@ -181,6 +181,56 @@ def _create_transplant(
 +adding one more line again
 """.strip()
 
+PATCH_FORMATTED_1 = r"""
+# HG changeset patch
+# User Test User <test@example.com>
+# Date 0 0
+#      Thu Jan 01 00:00:00 1970 +0000
+# Diff Start Line 7
+add another file.
+
+diff --git a/test.txt b/test.txt
+--- a/test.txt
++++ b/test.txt
+@@ -1,1 +1,4 @@
+ TEST
++
++
++adding another line
+""".strip()
+
+PATCH_FORMATTED_2 = r"""
+# HG changeset patch
+# User Test User <test@example.com>
+# Date 0 0
+#      Thu Jan 01 00:00:00 1970 +0000
+# Diff Start Line 7
+add another file.
+diff --git a/test.txt b/test.txt
+--- a/test.txt
++++ b/test.txt
+@@ -2,3 +2,4 @@ TEST
+
+ 
+ adding another line
++add one more line
+""".strip()
+
+TESTTXT_FORMATTED_1 = b"""
+TeSt
+
+
+aDdInG AnOtHeR LiNe
```

Comment:
I implemented this in an attempt to show `hg fix`'s behaviour of not touching unrelated changes in a patch. Turns out that behaviour is behind the `linerange` config option. After looking around it turns out that feature is supported by clang-format and prettier, but not by black. So black changes will always modify the entire file, including unrelated changes to the patch. This might end up being fine but worth pointing out now.